client/lib/products-values/index.js

```diff
@@ -359,6 +370,22 @@ export function getDomain( product ) {
 	return product.meta;
 }
 
+export function getProductsKeys() {
```

Comment:
Any more appropriate term than `key` here? Product slugs?

Or maybe we can directly use the constant where necessary? Although I don't mind having consistency with what we're doing for plans.

Reply:
I'd prefer to use a getter instead of accessing the constant directly.

Reply:
Sounds good to me 👍